Packs/Whois/ReleaseNotes/1_3_0.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Whois
+- Updated mapped fields **country** and **organization name** for ip indicator in **ip** command.
```

Comment:
```suggestion
- Fixed an issue where the **Country** and **Organization** outputs of the ***ip*** command were populated with the ASN data instead of the IP data.
```